api/app.js

```diff
@@ -0,0 +1,85 @@
+// This is the entry point for your add-on, creating and configuring
```

Comment:
Need to refactor this page. We may do it later.

Reply:
Yes later :smile: